src/topology/algebra/infinite_sum.lean

```diff
@@ -414,6 +414,24 @@ lemma tsum_congr_subtype (f : β → α) {s t : set β} (h : s = t) :
   ∑' (x : s), f x = ∑' (x : t), f x :=
 by rw h
 
+lemma tsum_zero' (hz : is_closed ({0} : set α)) : ∑' b : β, (0 : α) = 0 :=
```

Comment:
Note that `hz` is equivalent to `t2_space` for topological groups: https://leanprover-community.github.io/mathlib_docs/topology/algebra/uniform_group.html#topological_add_group.t2_space_iff_zero_closed

However, I think that `α` only has to be a monoid in this setting, so I guess this variant is still useful.

Reply:
Not only does it only have to be a monoid but there is no assumption relating the topology and algebra structure (not even `has_continuous_mul`)